### ISSUE TITLE
Allow changing server ports

### ISSUE THREAD

**Issue by KimchaC** [Place1/wg-access-server#27]
Another (low priority) feature request... I don't mean to overwhelm you with all of it or seem entitled. It just seems you are really motivated to make this a great project and I want to contribute from a user's perspective.

It would be great if we could adjust the web interface port from the config. For example, I had to run the server in network_mode=host where the server binds to the port on the host container instead of the docker container and had something else running on port 8000.

I had to change the port on my other existing service and all dependencies for it. It would be great if we could adjust that from the config.

**Comment by antoinebou13** [comment]
I don't know if that help, but
You can configure the webserver port in docker-compose.yml.
```
version: "3.7"
services:
  wg-access-server:
    image: place1/wg-access-server
    container_name: wg-access-server
    cap_add:
      - NET_ADMIN
    volumes:
      - "wg-access-server-data:/data"
    #   - "./config.yaml:/config.yaml" # if you have a custom config file
    ports:
      - "8005:8000/tcp"
      - "51820:51820/udp"
	# network_mode: host # or use network mode
    devices:
      - "/dev/net/tun:/dev/net/tun"

# shared volumes with the host
volumes:
  wg-access-server-data:
    driver: local
```
then run
```
docker-compose up -d
```

**Comment by Place1** [comment]
> I want to contribute from a user's perspective.

thanks, this is something that's easy to overlook when writing your own project.

i'll make a PR and an new RC with this change, should be easy.

**Comment by Place1** [comment]
@KimchaC this is now available in 0.2.0-rc2
